pcsx2/R3000AInterpreter.cpp

```diff
@@ -144,6 +144,12 @@ static __fi void execI()
 	psxRegs.cycle++;
 	iopCycleEE-=8;
 
+	if ((psxHu32(HW_ICFG) & (1 << 3)))
+	{
+		//One of the Iop to EE delta clocks to be set in PS1 mode.
+		iopCycleEE-=9;
+	}
```

Comment:
Is the total of -17 intended?

Reply:
Yes

Reply:
Can you explain why?

Reply:
@turteli because that adjusts the freaquency of IOP to EE communication ie. How many times the IOP converses with the EE and back. The normal ps2 setting is about 8 MHz whilst the ps1 is around 8.71 MHz but I set it to 9 because everything runs off of ints instead of floats.

Reply:
Immediately above the code you added is `iopCycleEE-=8;` You're subtracting another 9 here for a total of -17. So in PS2 mode it subtracts 8 and in PSX mode it subtracts 17. Are you sure that's right?

Reply:
I would say that isn't right. You've got the IOP running at about half the speed it should be.

Reply:
The if statement is for ps1 mode meaning it'll only run in ps1 mode, I can ad an else real quick if you'd like though.

Reply:
Yes but the PS2 mode one is running as well.  You should have an else statement for the PS2 one if that's the case.

Reply:
done

Reply:
Ehr, sorry, that's confusing me now.
The `iopCycleEE-=8` was always there and should be correct for PS2 mode.
The new change was to add another 9 cycles for when it is in PSX mode, for a total of 17 cycles, right?

However, it makes more sense now:
8 cycles for PS2 mode, 9 cycles for PSX mode.
@kenshen112 Can you confirm this is what it should be?

Reply:
@ramapcsx2 it is, infact by seperating the two the audio issues that were experienced with this build vanish. This PR is complete unless people have other issues?